src/wallet/external_signer_scriptpubkeyman.h

```diff
@@ -32,5 +31,3 @@ class ExternalSignerScriptPubKeyMan : public DescriptorScriptPubKeyMan
   TransactionError FillPSBT(PartiallySignedTransaction& psbt, int sighash_type = 1 /* SIGHASH_ALL */, bool sign = true, bool bip32derivs = false, int* n_signed = nullptr) const override;
 };
 #endif
-
-#endif // BITCOIN_WALLET_EXTERNAL_SIGNER_SCRIPTPUBKEYMAN_H
```

Comment:
I think you should keep the comment for style

Reply:
I'm surprised the linter didn't pick that up, fixed.